docs/yubikey_setup.rst

```diff
@@ -13,10 +13,12 @@ What is a YubiKey?
 ------------------
 
 A YubiKey is a physical token used for two-factor authentication. They
-are made by a company called Yubico and are `commercially available`_.
+are made by a company called Yubico and are `commercially available`_. Note that not all physical tokens are compatible with the YubiKey Personalization Tool; for this, you require `a key that can support OATH-HOTP`_.
```

Comment:
80-char lines here too

Reply:
sorry about all the >80-line chars. I have it set to wrap at 80 but evidently not playing nice.

Reply:
fixed in 1c7e1817a